internal/install/static_kubernetes_elastic_agent_yml.go

```diff
@@ -0,0 +1,125 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+const kubernetesDeployerElasticAgentYml = `---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kind-fleet-agent-clusterscope
+  namespace: kube-system
+  labels:
+    app: kind-fleet-agent-clusterscope
+    group: fleet
+spec:
+  selector:
+    matchLabels:
+      app: kind-fleet-agent-clusterscope
+  template:
+    metadata:
+      labels:
+        app: kind-fleet-agent-clusterscope
+        group: fleet
+    spec:
+      serviceAccountName: kind-fleet-agent
+      containers:
+        - name: kind-fleet-agent-clusterscope
+          image: docker.elastic.co/beats/elastic-agent:{{ STACK_VERSION }}
```

Comment:
IIUC, when the k8s deployer is used, we'll spin up a container in the k8s cluster for the Elastic Agent. Additionally, we will still have the Elastic Agent that is spun up by `elastic-package stack up`, right?

Reply:
Right, I'm starting an additional instance of the Elastic Agent (lazy initialization) and leaving it up there for next tests.